p2p/simulator/main.go

```diff
@@ -0,0 +1,186 @@
+}
+
+var messages = []string{
+	"hello friend!\n",
+	"i am a node\n",
+	"do you want to be friends?\n",
+	"ok\n",
+	"noot\n",
+	"i am a penguin stuck in a computer\n",
+	"pls feed me code\n",
+}
+
+func StartIpfsNode() (*ipfs.IpfsNode, error) {
```

Comment:
duplicate of p2p.StartIpfsNode, should we make it common(-ish)?

Reply:
Perhaps the real question here is whether to utilize this simulator inside of the p2p tests?

Reply:
maybe we should make a common p2p?

Reply:
i have moved all duplicate functions to p2p/common.go